src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -2928,6 +2936,45 @@ checkNullable:
             Return typeName
         End Function
 
+        Private Function ParseTupleType(openParen As PunctuationSyntax) As TypeSyntax
```

Comment:
Again - that is mostly for error recovery. I want to accept this grammatically, but then make it an error at binding. 
It could be a parsing error too. I would not have a lot of objections here. I just want to not give up parsing when I see `(Integer)`